Apps/MAPL_Component_Driver/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+esma_set_this (OVERRIDE root_gridcomp)
```

Comment:
This does not seem like the right name - too generic.  Will discuss when you return.

Reply:
ok - you'll update the PR?

Reply:
I'm thinking we may not even want a library here.   Maybe those few sources just go into the executable?   And then we can just leave the default name - has little role at that point.